python/QC3.py

```diff
@@ -0,0 +1,28 @@
+import datetime,os,sys
+import subprocess
+from subprocess import CalledProcessError
+from wrappers import runCommand
+from options import parser
+
+
+#print args.file
```

Comment:
remove lines of code which have been commented out, white space reduction needed.